### PR TITLE
Storing SP custom css

### PR DESCRIPTION
**Pull request by zeek01**
## What does this change?
This adds a place to store Sourcepoint CMP UI Custom CSS to capture changes and create a form of version history.

### CODE REVIEW

**Review by sndrs — APPROVED** [review]
great 👍

File: sourcepoint-ui/TCFv2_banner_amp.css [hunk 1]
```diff
@@ -0,0 +1,443 @@
+/*
+
+Please update the CMP repo with any launched changes
+https://github.com/guardian/consent-management-platform
```
Comment (mxdvl) [inline sourcepoint-ui/TCFv2_banner_amp.css:4]: I would suggest making this a little more explicit.

```suggestion
https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui
```

File: sourcepoint-ui/TCFv2_banner_app.css [hunk 2]
```diff
@@ -0,0 +1,436 @@
+/*
+
+Please update the CMP repo with any launched changes
+https://github.com/guardian/consent-management-platform
```
Comment (mxdvl) [inline sourcepoint-ui/TCFv2_banner_app.css:4]:
```suggestion
https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui
```

File: sourcepoint-ui/TCFv2_pm_app.css [hunk 3]
```diff
@@ -0,0 +1,354 @@
+/*
+
+Please update the CMP repo with any launched changes
+https://github.com/guardian/consent-management-platform
```
Comment (mxdvl) [inline sourcepoint-ui/TCFv2_pm_app.css:4]:
```suggestion
https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui
```

File: sourcepoint-ui/TCFv2_pm_amp.css [hunk 4]
```diff
@@ -0,0 +1,444 @@
+/*
+
+Please update the CMP repo with any launched changes
+https://github.com/guardian/consent-management-platform
```
Comment (mxdvl) [inline sourcepoint-ui/TCFv2_pm_amp.css:4]:
```suggestion
https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui
```

File: sourcepoint-ui/TCFv2_banner_web.css [hunk 5]
```diff
@@ -0,0 +1,441 @@
+/*
+
+Please update the CMP repo with any launched changes
+https://github.com/guardian/consent-management-platform
```
Comment (mxdvl) [inline sourcepoint-ui/TCFv2_banner_web.css:4]: https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui

File: sourcepoint-ui/TCFv2_pm_web.css [hunk 6]
```diff
@@ -0,0 +1,376 @@
+/*
+
+Please update the CMP repo with any launched changes
+https://github.com/guardian/consent-management-platform
```
Comment (mxdvl) [inline sourcepoint-ui/TCFv2_pm_web.css:4]:
```suggestion
https://github.com/guardian/consent-management-platform/tree/main/sourcepoint-ui
```

**Review by mxdvl — APPROVED** [review]
That’s great. I’ve updated the URL to point the the subfolder containing this information.

It might be worth adding a few images and the like in the `README.md` for future travellers.

**Comment by mxdvl** [comment]
@zeek01 / @ezekiel01 you can merge this!

**Pull request merged**